### ISSUE TITLE
New Retry times for not getting banned

### ISSUE THREAD

**Issue by rohitsakala** [uxDaniel/visa_rescheduler#29]
Can some suggest new times for not getting banned 

STEP_TIME = 0.5  # time between steps (interactions with forms): 0.5 seconds
RETRY_TIME = 60*10  # wait time between retries/checks for available dates: 10 minutes
EXCEPTION_TIME = 60*30  # wait time when an exception occurs: 30 minutes
COOLDOWN_TIME = 60*60  # wait time when temporary banned (empty list): 60 minutes

**Comment by Soroosh-N** [comment]
> Can some suggest new times for not getting banned
> 
> STEP_TIME = 0.5 # time between steps (interactions with forms): 0.5 seconds RETRY_TIME = 60_10 # wait time between retries/checks for available dates: 10 minutes EXCEPTION_TIME = 60_30 # wait time when an exception occurs: 30 minutes COOLDOWN_TIME = 60*60 # wait time when temporary banned (empty list): 60 minutes

After fixing the issue [myRep](https://github.com/Soroosh-N/us_visa_scheduler), I started running the code again, and it seems to be working well. However, it seems it bans the IP from time to time. It's a little unusual. I don't know if this is because of my IP or Internet connection. I will let you know if this problem keeps happening. By the way, I set the Cooldown time to be 5 hours, and the retry time can be every 5 minutes. I think it's good to let the program rest every 1.5 hours for like half an hour.
I hope it helps.

**Comment by rohitsakala** [comment]
@Soroosh-N does 1.5 hour rest work without banning ?

**Comment by afshanmomin-git** [comment]
what the best rety times to avoid bans?
I keep getting empty list (no dates) does that mean i am banned ?

**Comment by Soroosh-N** [comment]
> what the best rety times to avoid bans? I keep getting empty list (no dates) does that mean i am banned ?

Yes, when you get an empty list, it means you are temporarily banned from accessing the scheduling table. This temporary ban will take 5 hours. As I know, there is no way you can avoid the banning process. Companies use multiple accounts to check the table all day long. I think it's not possible for us to do something like that with one account.

**Comment by alamoudi10** [comment]
can we make multiple accounts using the same DS-160 form?
